packages/esm-patient-registration-app/src/patient-registration/field/address/address-field.component.tsx

```diff
@@ -23,42 +24,49 @@ export const AddressField: React.FC = () => {
   const { addressTemplate } = useContext(ResourcesContext);
   const [addressFields, setAddressFields] = useState([]);
   const { t } = useTranslation();
-  const addressTemplateXml = addressTemplate.results[0].value;
+  const addressTemplateXml = addressTemplate?.results[0].value;
 
   useEffect(() => {
-    const templateXmlDoc = parseString(addressTemplateXml);
-    const nameMappings = getTagAsDocument('nameMappings', templateXmlDoc);
-    const elementDefaults = getTagAsDocument('elementdefaults', templateXmlDoc);
-    const properties = nameMappings.getElementsByTagName('property');
-    const propertiesObj = Array.prototype.map.call(properties, (property: Element) => {
-      const name = property.getAttribute('name');
+    if (addressTemplateXml) {
```

Comment:
ewww... xml...

Reply:
Yes, the adress fields fetched comes in the XML format.